### ISSUE TITLE
allow to easily follow a spectator

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#2039]
This idea comes directly from «lautre» on IRC/matrix:

>  23:59:26 <+lautre[m]> Pour ceux qui ne savent pas, la commande /follow ici_nom_joueur  permet de suivre quelqu'un, même s'il suit quelqu'un d'autre.

translates to:

> for those who don't know, command `/follow PLAYER_NAME` allows to follow someone not playing

I think requiring console to do actions is bad, hence this issue.

**Comment by illwieckz** [comment]
Well, that's an advanced need, but may do a cvar to cycle among everyone and not just players.

**Comment by bmorel** [comment]
Yes, only reported it so that it stays around, as it's a pretty interesting idea imo.

**Comment by bmorel** [comment]
@superdupont-fr hopefully this works

**Comment by necessarily-equal** [comment]
The concept is interesting. It may suit my own needs because I'd like to add a feature so that the player following the jugg would keep following {the jugg} even if the player who is the jugg changes.

As for actually implementi'g what this UI is about, which is following someone with a GUI, I must say this isn't easy. It may happen once (if) I make the player list clickable.

**Comment by DolceTriade** [comment]
You can already cycle between players by right clicking as a spec, but I agree it's very non-intuitive. Previously you needed to press "enter" to spec but that was changed to chat recently.
